modules/dashboard/src/components/StatsSummary.js

```diff
@@ -8,39 +8,56 @@ const styles = {
     flexWrap: "wrap",
     flexDirection: "row",
     width: "100%",
-    height:"100%",
+    height: "100%",
     justifyContent: "center",
     alignItems: "center",
   },
   card: {
-    marginTop:"15%",
-      display:"flex",
-      height:"320px",
-      width:"320px",
-      alignItems:"center",
-      justifyContent:"center",
-      margin:"0% 2% 0% 2%",
-      border: "3px solid #002868",
-      textDecoration:"none",
-      '&:hover': {backgroundColor:"rgba(0,40,104,0.2)"}
+    marginTop: "15%",
```

Comment:
Can we use the styled-components syntax since this is new code?

Reply:
yeah -- will incorporate w/ next set of changes